automan/cluster_manager.py

```diff
@@ -368,6 +377,26 @@ def update(self, rebuild=True):
                 if rebuild:
                     self._rebuild(host, home)
 
+    def delete(self, sim_dir, remotes):
+        if remotes[-1] == 'all':
+            hosts = [h.get('host') for h in self.workers]
+            remotes = hosts.copy()
+        else:
+            for remote in remotes:
+                hosts = [h.get('host') for h in self.workers]
```

Comment:
Why is this in the for loop?  the workers don't change in this scope.

Reply:
fixed.